Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -460,10 +460,14 @@ public struct TraversalOptions {
   /// - map fields: Will always be visited, even if they are empty.
   /// - singular message and group fields: Will be visited only if they are set.
   /// - singular primitive fields: Will always be visited under proto2. Under proto3 unset optional primitive fields will not be visited.
-  public var visitDefaultValues: Bool
+  public var alwaysVisitPrimitiveFields: Bool
+    
+    public var visitDefaultValues: Bool {
```

Comment:
Why include this?

Reply:
oops, left over from a refactor. Will delete.